modules/nf-core/fastqc/main.nf

```diff
@@ -26,7 +26,10 @@ process FASTQC {
     def rename_to = old_new_pairs*.join(' ').join(' ')
     def renamed_files = old_new_pairs.collect{ old_name, new_name -> new_name }.join(' ')
 
-    def memory_in_mb = MemoryUnit.of("${task.memory}").toUnit('MB')
+    // The total amount of allocated RAM by FastQC is equal to the number of threads defined (--threads) time the amount of RAM defined (--memory)
```

Comment:
Why is that fix part of thiw PR which seems dedicated to CI-testing?

Reply:
Just an update of FASTQC to trigger tests, I chose this module because changes were meaningless